diff --git a/CMakeLists.txt b/CMakeLists.txt
index d4ea05e2..23a6fcc6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -187,8 +187,8 @@ else()
     set(MIOPEN_CACHE_DIR "~/.cache/miopen/" CACHE STRING "")
 endif()

-set(CPACK_DEBIAN_PACKAGE_DEPENDS "openssl, rocm-opencl-dev, rocm-utils, hip_hcc, miopengemm")
-set(CPACK_RPM_PACKAGE_REQUIRES "openssl, rocm-opencl-devel, rocm-utils, hip_hcc, miopengemm")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "openssl, rocm-opencl-dev, hip_hcc, miopengemm")
+set(CPACK_RPM_PACKAGE_REQUIRES "openssl, rocm-opencl-devel, hip_hcc, miopengemm")

 rocm_create_package(
     NAME MIOpen-${MIOPEN_BACKEND}
